### PR TITLE
Modified ubuntu jdk7 image to tomcat:7-jre7-alpine (Rebased)

### LINKED ISSUE #3: Replace manishkumr/ubuntu_jdk7_image base tomcat7 base container
It would be good to replace the manishkumr/ubuntu_jdk7_image base with a new image built from a dockerfile

### PR DESCRIPTION
**Pull request by johnflavin**
This is a cleaned-up version of PR #6, rebased on master.

Closes #3 
Closes #6

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,50 +1,96 @@
...
+      - /var/run/docker.sock:/var/run/docker.sock
+    links:
+      - xnat-db
+    environment:
+      - XNAT_DATASOURCE_DRIVER=org.postgresql.Driver
+      - XNAT_DATASOURCE_URL=jdbc:postgresql://xnat-db/xnat
+      - XNAT_DATASOURCE_USERNAME=xnat
+      - XNAT_DATASOURCE_PASSWORD=xnat
+      - XNAT_HIBERNATE_DIALECT=org.hibernate.dialect.PostgreSQL9Dialect
+<<<<<<< HEAD
```
Comment (Sikerdebaard) [inline docker-compose.yml]: Pretty sure this doesn't belong here :sweat_smile:

File: docker-compose.yml [hunk 2]
```diff
@@ -1,50 +1,96 @@
...
+#        volumes:
+#          - /:/rootfs:ro
+#          - /var/run:/var/run:rw
+#          - /sys:/sys:ro
+#          - /var/lib/docker/:/var/lib/docker:ro
+#        ports:
+#        - "8082:8080"
+#        expose:
+#          - 8082
+=======
```
Comment (Sikerdebaard) [inline docker-compose.yml]: Pretty sure this doesn't belong here :sweat_smile:

**Comment by johnflavin** [comment]
Ok, I think I have fixed that problem.

**Comment by johnflavin** [comment]
No, never mind. I have to open another PR. 😞

**Comment by manishkumr** [comment]
I was under impression that XNAT requires JDK instead of JRE, and this was one of the reason to use pre-built ubuntu-JDK-7 image. May be I have missed something.. Does this [documentation](https://wiki.xnat.org/documentation/getting-started-with-xnat-1-7/installing-xnat-1-7/prerequisites-for-installing-xnat-1-7) still valid ?

**Pull request closed without merging**